package/batocera/emulationstation/batocera-es-system/es_systems.yml

```diff
@@ -1040,6 +1040,18 @@ lightgun:
           Un exemple de setup qui fonctionne est si vous utilisez une DolphinBar en 'mode 2' (mode jeu clavier/souris) avec une Wiimote comme flingue.
           Si vous avez plus d'une souris installee, assurez-vous que la DolphinBar / Wiimote soit reconnue comme une des deux premieres.
 
+ports:
+  name:       Ports
+  extensions: [sh]
+  group: ports
+  path:       /userdata/roms/ports
+  command:    "%ROM%"
```

Comment:
this command will fail on vFAT because there is no executable bit. Or if the user forgots to set this.
Therefore I'd suggest to call shell commands by the bash interpreter `bash %ROM%`

Reply:
yes on fat 32 many problems ... on fat 32 Amiberry is not working i discover it today, because LN link to bios files are not working. Not a problem on EXT4 and NTFS.
But for sure ports and homebrew will be on userdata/roms/ directory ... and we must warn the user to not use fat32 for this.

Reply:
Good catch, guys. Just proposed a fix. 
And we should also tell everyone to stop using FAT32 or extFAT....

Reply:
> And we should also tell everyone to stop using FAT32 or extFAT....

btrfs :)